### ISSUE TITLE
fail to translate

### ISSUE THREAD

**Issue by jerry20130912** [ssut/py-googletrans#96]
after install googletrans, I use following commands to translate, but fails:
[root@test googletrans]# translate -c "你好"
Traceback (most recent call last):
  File "/bin/translate", line 40, in <module>
    main()
  File "/bin/translate", line 21, in main
    result = translator.detect(args.text)
  File "/usr/lib/python2.7/site-packages/googletrans/client.py", line 249, in detect
    data = self._translate(text, dest='en', src='auto')
  File "/usr/lib/python2.7/site-packages/googletrans/client.py", line 75, in _translate
    token = self.token_acquirer.do(text)
  File "/usr/lib/python2.7/site-packages/googletrans/gtoken.py", line 180, in do
    self._update()
  File "/usr/lib/python2.7/site-packages/googletrans/gtoken.py", line 59, in _update
    code = unicode(self.RE_TKK.search(r.text).group(1)).replace('var ', '')
AttributeError: 'NoneType' object has no attribute 'group'

**Comment by su27k-2003** [comment]
I had the same error but elzeard91's branch [here](https://github.com/elzeard91/py-googletrans/blob/8940fb67f9178aad535b786b2af411e78c5f83c7/googletrans/gtoken.py) fixed the error perfectly. you just need to overwrite the gtoken.py with elzeard91's one.

**Comment by dsc03** [comment]
Is elzeard91's version going to be pushed in version 2.3.1/2.4.0? Having to overwrite makes it a bit more difficult to run on a remote server (such as ones provided by AWS) because it won't install elzeard91's version when installing requirements.txt packages

**Comment by aradove** [comment]
How do I add it? I only find the .egg files for googletrans.

**Comment by Gorialis** [comment]
On a machine with `git` installed you can install elzeard91's fixed branch using
```sh
pip install -U git+https://github.com/elzeard91/py-googletrans@8940fb67f9178aad535b786b2af411e78c5f83c7#egg=googletrans
```
without needing to download the files or edit anything manually.

**Comment by ssut** [comment]
@dsc03 @aradove @Gorialis I released 2.4.0 on PyPI! sorry for the late.
